content/docs/demos/outbound_ip_exclusion.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Outbound Traffic IP Range Exclusions"
+description: "Excluding IP address ranges of outbound traffic from sidecar interception"
+type: docs
+weight: 2
+---
+
+This guide demonstrates how outbound IP address ranges can be excluded from being intercepted by OSM's proxy sidecar, so as to not subject them to service mesh filtering and routing policies.
+
+## Prerequisites
+
+- Kubernetes cluster running Kubernetes v1.19.0 or greater.
+- Have OSM installed.
+- Have `kubectl` available to interact with the API server.
+- Have `osm` CLI available for managing the service mesh.
+
+
+## Demo
+
+The following demo shows an HTTP `curl` client making HTTP requests to the `httpbin.org` website directly using its IP address. We will explicitly disable the egress functionality to ensure traffic to a non-mesh destination (`httpbin.org` in this demo) is not able to egress the pod.
+
+1. Disable mesh-wide egress passthrough.
```

Comment:
Instead of asking the user to disable egress, should we reword to check if egress is disabled ?

Reply:
No, egress should be disabled, and if it is already disabled the command will be a no-op, which is okay.